.github/workflows/end-to-end.yml

```diff
@@ -1,6 +1,7 @@
 name: End-to-End tests
 
 on:
+  push:
```

Comment:
This change is only to simplify testing. I'll revert this after PR is approved, before merging.